### PR TITLE
hotfix for influx auth create

### PR DESCRIPTION
**Pull request by kelseiv**
hotfix for `influx auth create` issue
https://influxdata.slack.com/archives/CPYJNHM4K/p1665607451322279

@jstirnaman @sanderson --please let me know if we need to update and/or add this workaround elsewhere in the docs. thank you!

### CODE REVIEW

File: content/influxdb/v2.4/reference/cli/influx/auth/create.md [hunk 1]
```diff
@@ -11,6 +11,64 @@ updated_in: CLI v2.3.0
...
+
+{{% warn %}}
+InfluxDB 2.4 introduced a bug that prevents you from creating an **all-access** or **operator** token using the `influx auth create` command, and causes the following error: `Error: could not write auth with provided arguments: 403 Forbidden: permission.`
+
+Until this bug is resolved in the next influx CLI release, please use the [workaround below to create an all-access or operator token](/influxdb/v2.4/security/tokens/create-token/#workaround-to-create-an-all-access-or-operator-token).
+{{% /warn %}}
+
+### **Workaround:** To create an all-access or operator token
+
+- Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an **operator** token, you must also include the `--read-orgs` and `--write-orgs` flags.
```
Comment (sanderson) [inline content/influxdb/v2.4/reference/cli/influx/auth/create.md:23]:
```suggestion
Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an **operator** token, you must also include the `--read-orgs` and `--write-orgs` flags.
```

File: content/influxdb/v2.4/security/tokens/create-token.md [hunk 2]
```diff
@@ -111,6 +111,64 @@ find the token you want to clone and click the **{{< icon "settings" >}}** icon
...
 
+{{% warn %}}
+InfluxDB 2.4 introduced a bug that prevents you from creating an **all-access** or **operator** token using the `influx auth create` command, and causes the following error: `Error: could not write auth with provided arguments: 403 Forbidden: permission.`
+
+Until this bug is resolved in the next influx CLI release, please use the [workaround below to create an all-access or operator token](/influxdb/v2.4/security/tokens/create-token/#workaround-to-create-an-all-access-or-operator-token).
+{{% /warn %}}
+
+### **Workaround:** To create an all-access or operator token
+
+- Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an operator token, you must also include the `--read-orgs` and `--write-orgs` flags.
```
Comment (sanderson) [inline content/influxdb/v2.4/security/tokens/create-token.md:122]:
```suggestion
Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an operator token, you must also include the `--read-orgs` and `--write-orgs` flags.
```

**Review by sanderson — APPROVED** [review]
Two suggestions. The content under the new heading doesn't need to be a bulleted list.

File: content/influxdb/v2.4/reference/cli/influx/auth/create.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/v2.4/reference/cli/influx/auth/create.md:23]: Thank you, @sanderson--added bullet per [style here](https://developers.google.com/style/procedures#single-step-procedures) and bc I think it looks more crisp :-)

**Pull request merged**